src/main/scala/chisel3/iotesters/ChiselSpec.scala

```diff
@@ -10,14 +10,14 @@ import chisel3.testers._
 
 /** Common utility functions for Chisel unit tests. */
 trait ChiselRunners extends Assertions {
+  val backends = Seq("firrtl", "verilator", "vcs")
```

Comment:
There should be some test to verify that vcs is available (possibly steal the code from chisel2), otherwise, all the tutorials fail on systems that don't have vcs installed.